src/pages/Extraction.tsx

```diff
@@ -46,13 +48,34 @@ function buildExtractionTableData(ctx: ExtractionContext) {
     .flat();
 }
 
-function Extraction() {
-  const [current, send] = useMachine(() => extractionMachine.withContext({ labwares: [], workNumber: '' }));
+/***
+ * Equipments with 'extract' category
+ */
+type ExtractionEquipments = {
+  equipments: EquipmentFieldsFragment[];
+};
+
+const extractionEquipments = (equipments: EquipmentFieldsFragment[]) => {
+  const methods: Array<{ value: number; label: string }> = [];
+
+  equipments.forEach((equipment) => {
+    methods.push({ value: equipment.id, label: `Automated - ${equipment.name}` });
```

Comment:
If not confirmed already, it is good to communicate to users that 'Automated"  is appended to all new equipment ids in config page. This helps to avoid 'Automated' as a part of equipment name when new ones are added by them in config page

Reply:
hmm, I think it is quite confusing to add an equipment with 'Automated' as prefix also, as that will not much the way the other equipment'category

Reply:
Probably, it can added as a note in config page, in equipments tab for this category that "Automated" will be prefixed , so that it will be conveyed properly

Reply:
oh do mean we prefix the added equipments in the config page, I mean the user added a machine 'EZ2' and we stores as 'Automated - EZ2' ?

Reply:
No, I meant just to give a textual indication to the user in config page that whatever machine they will be added will be displayed with "Automated" prefix in the page. It is, just a thought, but probably , it can be tested by user and see how they think about it.

Reply:
If the user thinks it's ok, then it's perfectly fine